study-builder/studies/osteo/study.conf

```diff
@@ -1631,6 +1670,36 @@
       "dispatchToHousekeeping": false,
       "order": 3
     },
+    {
+      "label": "CREATE_FH_ON_CONSENT_COMPLETE",
```

Comment:
Leveraging event label feature added to study builder so we can add just these specific events.